source/includes/write/update.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+
+int
+main (void)
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_init ();
+
+    client = mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "sample_restaurants", "restaurants");
+
+    {
+        // Updates the "name" value of a document from "Bagels N Buns" to "2 Bagels 2 Buns"
+        // start-update-one
+        bson_t *query = BCON_NEW ("name", BCON_UTF8 ("Bagels N Buns"));
+        bson_t *update = BCON_NEW ("$set", "{", "name", BCON_UTF8 ("2 Bagels 2 Buns"), "}");
+        bson_error_t error;
+
+        if (!mongoc_collection_update_one (collection, query, update, NULL, NULL, &error)) {
+            fprintf (stderr, "Update one operation failed: %s\n", error.message);
+        }
```

Comment:
```suggestion
        }
        
```
Space in between the command and `destroy`s for consistency